auth-web/src/store/modules/business.ts

```diff
@@ -36,7 +37,7 @@ export default class BusinessModule extends VuexModule {
   }
 
   @Action({ rawError: true })
-  public async syncBusinesses (): Promise<void> {
+  public async syncBusinesses (enableSpGpDba: boolean): Promise<void> {
```

Comment:
Is this parameter something we want to require for every call of `syncBusinesses`?
If not, you can set a default here, `syncBusinesses (enableSpGpDba: boolean = false)..` so that is isn't required everywhere.